lab-davide/hello.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+const greet = require(__dirname + '/lib/greet');
```

Comment:
Its great to see that you named this variable the same as the modules file name... when programs get really complex this is a good pratice because it makes it easy to figure out where things come from :)